### PR TITLE
New launchers to nav and follow nav with small objects

### PR DESCRIPTION
**Pull request by Juancams**
It is still necessary to adjust parameters to navigate with slam following the person.

### CODE REVIEW

File: robocup_bringup/params/tiago_nav_params.yaml
Comment (juandpenan) [inline robocup_bringup/params/tiago_nav_params.yaml:1]: I would remove the the keep out filter line 199

File: robocup_bringup/params/tiago_nav_follow_params.yaml
Comment (juandpenan) [inline robocup_bringup/params/tiago_nav_follow_params.yaml:1]: the same, here remove the keep out filter

File: robocup_bringup/params/tiago_nav_follow_params.yaml [hunk 1]
```diff
@@ -396,11 +320,69 @@ velocity_smoother:
...
+    link_scan_maximum_distance: 1.5
+    loop_match_maximum_variance_coarse: 3.0
+    loop_match_minimum_chain_size: 10
+    loop_match_minimum_response_coarse: 0.35
+    loop_match_minimum_response_fine: 0.45
+    loop_search_maximum_distance: 3.0
+    loop_search_space_dimension: 8.0
+    loop_search_space_resolution: 0.05
+    loop_search_space_smear_deviation: 0.03
+    map_file_name: /home/juanca/ros/coresense_ws/install/robocup_bringup/share/robocup_bringup/maps/lab_robocup
```
Comment (juandpenan) [inline robocup_bringup/params/tiago_nav_follow_params.yaml:361]: Can we change this to only the name ?

File: robocup_bringup/thirdparty.repos [hunk 2]
```diff
@@ -6,7 +6,7 @@ repositories:
   ThirdParty/small_objects_detector:
     type: git
     url: https://github.com/gentlebots/small_objects_detector.git
```
Comment (juandpenan) [inline robocup_bringup/thirdparty.repos:8]: there is a namespace by default the makes navitation to subscribe to a different topic

Comment (Juancams) [reply robocup_bringup/thirdparty.repos:8]: Done

File: robocup_bringup/params/tiago_nav_follow_params.yaml (hunk 1 above)
Comment (Juancams) [reply robocup_bringup/params/tiago_nav_follow_params.yaml:361]: No

File: robocup_bringup/params/tiago_nav_params.yaml
Comment (Juancams) [reply robocup_bringup/params/tiago_nav_params.yaml:1]: Done

File: robocup_bringup/params/tiago_nav_follow_params.yaml
Comment (Juancams) [reply robocup_bringup/params/tiago_nav_follow_params.yaml:1]: Done

**Pull request merged**